src/lib/components/BrandingFooter.tsx

```diff
@@ -0,0 +1,40 @@
+import { Trans } from '@lingui/macro'
+import Row from 'lib/components/Row'
+import { Logo } from 'lib/icons'
+import styled from 'lib/theme'
+
+import ExternalLink from './ExternalLink'
+
+const UniswapA = styled(ExternalLink)`
+  color: ${({ theme }) => theme.secondary};
+  cursor: pointer;
+  display: flex;
+  font-size: 0.75em;
```

Comment:
Font size and line height should come from a ThemedText element and not specified explicitly in order to keep styles consistent and semantically named.

Reply:
(If there isn't an existing ThemedText component to match Phil's spec, you should make/name a new one for branding.)